src/components/BottomBar/Controls.styl

```diff
@@ -18,6 +18,42 @@
       &_look_danger
         border-color #e1e0e2
 
+    .submit
+      border-radius 4px
+      background #09F
+      box-shadow 0px 4px 4px 0px rgba(0, 0, 0, 0.25)
+      display flex
+      width 160px
+      height 40px
+      justify-content space-between
+      align-items center
+      color var(--primary-palette-light-theme-on-primary, #FFF)
```

Comment:
where is this css var defined? because I like the idea of moving in this direction of defining colour constants

Reply:
was trying to be future thinking but yea, its better not to assume, deleting